uber/models.py

```diff
@@ -2130,8 +2156,8 @@ def _attribute_adjustments(self):
         if not self.uses_allowed:
             self.uses_allowed = None
 
-        # If 'discount' is empty, then this is a full discount, free badge
-        if not self.discount:
+        # If this is a full discount, free badge, then nullify 'discount'
+        if self.is_free:
```

Comment:
Hm, will this not cause confusing behavior for admins? I know if I set a percent discount to `100` and had it reset to 0, I'd be confused. I'm not seeing it in action, of course -- maybe it all displays just fine?